src/App.tsx

```diff
@@ -1,7 +1,13 @@
 import JGButton from "./components/buttons/JGButton";
+import ImageUploader from "./components/uploads/ImageUploader";
 
 function App() {
-  return <JGButton>jgjgill</JGButton>;
+  return (
+    <>
+      <JGButton>jgjgill</JGButton>
+      <ImageUploader></ImageUploader>
```

Comment:
이렇게 사용하는 형태로 설명해도 좋을 것 같아요..!

```suggestion
      <ImageUploader />
```